behat/features/tabs.feature

```diff
@@ -0,0 +1,144 @@
+Feature: Book page
+  As an anonymous user
+  We need to be able to see the tabs list in the book page.
+
+  @api
+  Scenario Outline: Visit a books page
```

Comment:
Here it is - http://docs.behat.org/en/stable/guides/1.gherkin.html#tables

So the rule of thumb is - use a ``Scenario Outline`` only when the URL changes, otherwise use the tables example